### PR TITLE
Review code of youtube-frontpage

### PR DESCRIPTION
**Pull request by jacobo12montes**
-Add the navbar
-The main contains and the sidebar for recommended videos 
-Add the footer

### CODE REVIEW

**Comment by uryelah** [comment]
Hello **Juan Hincapie Montes** & **James Omage**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**First of all, welcome to Microverse and good job with your first project!🌟**

Good job adding semantic HTML to the project and spacing the elements properly. You just need to work on a couple of issues before moving on so keep up the good work 💪

---------    

## Required Changes 🏗 
 
- [x] Make sure your HTML file has no errors or warnings [at the validator.](https://validator.w3.org/)

- [x] If multiple page elements have the same CSS properties, ex: font-family, try to change them all in in one selector or make a specific class for the repeated style.
Remember to[ keep your code DRY.](https://meiert.com/en/blog/dry-css/)
EX:
`
/*Bad*/
.a {
  font-family: Arial, Helvetica, sans-serif;
}
.c {
  font-family: Arial, Helvetica, sans-serif;
}

/*Good*/
.a, .c {
  font-family: Arial, Helvetica, sans-serif;
}
`
- [x] The share button should be a `<button>` element.

- [x]  [!important] **Never use screenshots for icons or images**, if you need an icon for your project look for one at sites like [fontAwesome](https://fontawesome.com/), and images can easily be downloaded by right click 'view' image and then getting its path or downloading it. 
Using screenshots looks bad as the images have lower resolution, the icons do not scale properly and may have unwanted background colors if they are from screenshots.
Look how your icons under the main video have grey background colors while the actual background is white.

Screenshot_2020-03-30 youtube frontpage


- [x] Use actual icons for the navbar and under the main video.

- [x] Use images with good resolution for the suggested videos on the right, not screenshots


----

- [ ] [OPTIONAL] Change the color and text-decoration of the page links from the default blue to the color used by Youtube. 
**Your links**
Screenshot_2020-03-30 youtube frontpage(2)

**Original links:**
Screenshot_2020-03-30 Joyner Lucas - Will (ADHD)
----

- [x] [OPTIONAL] Add `cursor: pointer` property to all the clickable buttons in the page for better user experience.

- [ ] [OPTIONAL] If you are going to add a footer make sure to style it the same as the original footer, or as close as possible.
----

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by biodunJohnson** [comment]
completed the review code

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 🐱

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,187 @@
+* {
+    padding: 0;
+    margin: 0;
+}
```
Comment (mikenath223) [inline css/style.css:4]: - [x] You are attempting here to reset your browser defined CSS. To do this consider going to this [link](https://github.com/necolas/normalize.css/) and downloading the Normalize.css file and including it in your project.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Good Job Team 🎉👏

You did well for a first project. 🙂

However, there are a few changes that need to be made before moving forward.

**KINDLY READ EACH OF THE BELOW POINTS CAREFULLY AND FIX THE ISSUES POINTED OUT👇**

- [x] Required - Do ensure you are making your clone from this [link](http://archive.today/Bss88), as referenced under the Microverse Adjustment [section](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/) in Partwright.

- [x] Required - As seen in the link above your clone is expected to have a footer section. Kindly add this section.[Screenshot]👇
ice_screenshot_20200402-015031

- [x] Required - Also in the section just below the video template looks quite different from the original webpage. Kindly fix this.

**THIS IS YOURS👇**
ice_screenshot_20200402-014128

**THIS IS WHAT IT IS SUPPOSED TO BE👇**
ice_screenshot_20200402-015203

- [x] Required - Remove all inline code comments in your HTML code. If you want to refer to a previous version of your code use git history.

- [x] - Remove the HTML file from the subdirectory called `html` and simply place it in the root directory, and not in a folder.

**KINDLY FIX THE ABOVE STATED ISSUES👆 BEFORE REQUESTING ANOTHER REVIEW**

Welcome to Microverse🎉

**You did good for a first project**👏

Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Comment by jacobo12montes** [comment]
Hi @mikenath223 we just finished our project and change the next items
-We add a footer exactly as the example page
-We modify the section below the video, so it looks exactly like the example page

**Comment by Dandush03** [comment]
### :heavy_check_mark: Project Approve :boom:  :checkered_flag:
# Hey :hand: 
### Great Job so Far :grey_exclamation: :grey_exclamation: :muscle: :smiley: 

### Following Previous Reviews and Seeing you Have Made Required Changes. 
### The project Looks Great.:grey_exclamation::wink:

## [_Optional_]
* [ ] You should not have a folder named HTML and inside it the index, try putting it on the main folder (The root folder) :wink:

### Good Luck in your Next Assigment
### Happy coding:computer: 
#### Daniel Laloush

**Pull request merged**